ccdproc/image_collection.py

```diff
@@ -58,14 +58,18 @@ class ImageFileCollection(object):
         The filenames are assumed to be in ``location``.
         Default is ``None``.
 
+     ext: str or int, optional
+         The extension from which the header and data will be read in all files.
```

Comment:
I think you could drop the "in all files" part (simililarly in the property description). However maybe someone else wants to chime in here because I'm not sure if that part clarifies or confuses more.